README.md

```diff
@@ -1,6 +1,31 @@
 [![CI Status](https://github.com/AY2021S1-CS2103T-F11-3/tp/workflows/Java%20CI/badge.svg)](https://github.com/AY2021S1-CS2103T-F11-3/tp/actions)
 
-### User stories
+![Ui](docs/images/Ui.png)
+
+## Project scope
+  
+* The project simulates an ongoing software project for a desktop application (called _ProductiveNUS_).
+* ProductiveNUS provides an accessible platform for NUS computing students to manage and keep track of their 
+academic tasks at hand.  
+  * It is **written in OOP fashion**. It provides a **reasonably well-written** code base **bigger** (around 6 KLoC) 
+  than what students usually write in beginner-level SE modules, without being overwhelmingly big.
+  * It comes with a **reasonable level of user and developer documentation**.
+## Motivation
+* As a NUS Computing Student, you would probably have had **difficulties managing your
+academic schedule**. You often feel overwhelmed due to 
+**tight deadlines** for your assignments along with a **high workload** which you are on the verge of being unable to 
+cope with. In events like this, having a **disorganised timeline** makes your
+situation worse. You often **underestimate the time
+it takes** to complete your assignments and you might be **late for your lessons** as you are preoccupied 
+with completing them. However, all these
+tasks still remain uncompleted. You keep spinning your wheels.
```

Comment:
I think this is a good improvement from Minh's initial version. It is clearer and more straightforward.